content-src/components/LinkMenuButton/LinkMenuButton.js

```diff
@@ -4,7 +4,6 @@ const LinkMenuButton = React.createClass({
   render() {
     return (<button className="link-menu-button" onClick={e => {
       e.preventDefault();
-      e.stopPropagation();
```

Comment:
we should revert the changes here https://github.com/mozilla/activity-stream/pull/1748/files#diff-bea2b5a6e0bdde5d8bb69333c83d78e4 if we remove the stopPropogation here. Or else we get the bug #1372